robocopkaka/bootcamp-29-project
# #158254148 Show confirmation dialog on delete

#### What does this PR do?
show a confirmation dialog before deleting an event
#### Description of Task to be completed?
* show a confirmation dialog before deleting an event
* mock window.confirm globally
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install all dependecies
* run `npm run start:all` to start up the application
* go to `localhost:8000` on your browser
* click on `Signup`
* enter your details
* click on `Signup`
* request for admin access
* click on `Centers` on the navbar
* click on the red add button at the bottom right corner
* enter the details of the center
* click on Add Center
* add at least ten more centers
* click on `Centers`
* select a center
* click on the red add button at the bottom right corner
* enter the details of the event
* click on `Add Event`
* you should see the newly created event on the center's page
* click on the delete icon
* you should see a confirmation dialog at the top of the page
* click on `Ok`
* the center should be deleted
#### Any background context you want to provide?
wasn't previously showing a confirmation dialog or status message when deleting events
#### What are the relevant pivotal tracker stories?(if applicable)
#158254148
